curation/models.py

```diff
@@ -34,10 +36,16 @@ def get_for_model(self, model, for_concrete_model=True):
             # Load or create the ContentType entry. The smart_unicode() is
             # needed around opts.verbose_name_raw because name_raw might be a
             # django.utils.functional.__proxy__ object.
+            defaults = (
+                None
+                if IS_DJANGO_GTE_1_11
+                else {'name': smart_unicode(opts.verbose_name_raw)}
```

Comment:
Should this become `smart_text`? I thought `smart_unicode` was a deprecation warning in 1.11

Reply:
For Django 2 / Python 3 upgrade, we will need to have a whole sprint devoted to text/unicode issues. I have chosen not to devote any energies to these issues because the scope of the upgrade is already quite huge.

Reply:
Update: there is no actual deprecation warning given in Django 1.11, `smart_unicode` is just an alias for `smart_text`

Reply:
Yeah, I just double-checked that myself. 👍